src/app/modals/modals.component.ts

```diff
@@ -46,6 +46,7 @@ export class ModalsComponent implements DoCheck, OnInit {
   syncPercentage: number = 0;
   syncString: string;
   closeOnEscape: boolean = true;
+  disableClose: object; // @TODO CREATE model for object
```

Comment:
Don't use negative variable names, it should be 
enableClose 
Also not sure why it's an object?